presets/4.4/tune/karate/karate_race_lite.txt

```diff
@@ -107,25 +111,12 @@ set tpa_breakpoint = 1250
 
 #$ OPTION BEGIN (UNCHECKED): Dynamic idle for 4s 2850-3000kv
     #dynamic idle for 4s 2850-3000kv
-   set dyn_idle_min_rpm = 50
-   set dyn_idle_p_gain = 35
+	set dyn_idle_min_rpm = 50
+	set dyn_idle_p_gain = 35
```

Comment:
Please don't use tab
```suggestion
    set dyn_idle_min_rpm = 50
    set dyn_idle_p_gain = 35
```
```suggestion
	set dyn_idle_min_rpm = 50
	set dyn_idle_p_gain = 35
```

Reply:
I’m confused!! This is the old file I only changed the pid settings and as I pointed out to @limonspb  my editor is formatting stuff to it all looks fine..  when it’s not 😭 if I don’t use tab to indent stuff what should I use the space bar ?

Reply:
Depending on editor you are using there should be a setting to convert tabs into 4 spaces.

Reply:
> I didn’t rifle through the rest of the file sorry it’s a carry over from the old version

old file didn't have tabs :)

Reply:
k now have my editor showing me what's going on